_pages/en_US/installing-unlaunch.md

```diff
@@ -7,18 +7,20 @@ redirect_from:
 
 {% include toc title="Navigation" %}
 
-Unlaunch is an exploit that allows homebrew applications to take full advantage of the Nintendo DSi hardware at boot. Examples include dumping a Slot-1 device, launching into hiyaCFW amongst others.
+Unlaunch is an exploit that allows homebrew applications to take full advantage of the Nintendo DSi hardware at boot. Examples include dumping a Slot-1 device and launching into hiyaCFW, amongst others.
 
 {% capture usageInstructions %}
 WARNING: While low, there is a chance of **bricking your console** when installing Unlaunch. If you are worried whether you'd brick, these are all the benefits your console will gain by installing Unlaunch:
 
 - Higher compatibility with running old Nintendo DS homebrew via nds-bootstrap-hb (including the SNES emulator in TWiLight Menu++)
+- Better sound in GBARunner2
 - Launching incompatible flashcarts
-- Dumping SLOT-1 data
-- Launching DSiWare
+- Removing region locks on DSi enhanced / exclusive games
+- Accessing Slot-1 data
```

Comment:
Same issue as before, but why not just say "dump cartridges"?
or is what is meant here the ability to access files on the SD card of the flashcard?

Reply:
You can also dump and inject save files is why I changed it to access